README.md

```diff
@@ -1,2 +1,17 @@
 # InteractiveProgramming
 This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+
```

Comment:
For future reference, markdown uses double returns for proper spacing. So when you want a new line, hit enter twice.